msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting app extensibility
+author: laujan
+description: Understand the meeting app extensibility 
+ms.topic: conceptual
+---
+
+# Meeting app extensibility
+
+Teams’ meeting app extensibility is based on the following concepts:
+
+✔ Meeting lifecycle has stages such as before, during, and after meeting time frame.  
+✔ Participant roles in a meeting such as meeting organizer, presenter, or attendee.  
+✔ User types in a meeting such as in-tenant, guest, federated, or anonymous Teams user.
+
+This article covers information about the meeting lifecycle and how to integrate tabs, bots, and messaging extensions in your meeting. It also enables you to identify participant roles and also use different user types to perform tasks.
+
+> [!NOTE]
+> To work with the meeting app extensibility features, you must have the appropriate permissions.
+
+## Meeting lifecycle
+
+Meeting lifecycle consists of pre-meeting, in-meeting, and post-meeting app experience. You can integrate tabs, bots, and messaging extensions in each stage of the meeting lifecycle.
+
+### Integrate tabs into the meeting lifecycle
+
+Tabs allow team members to access services and content in a specific space within a channel or chat. This lets the team work directly with tabs and have conversations about the tools and data available within tabs. In Teams meeting, users can add a tab by selecting plus <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>, and choosing the app that they want to install as a tab.
```

Comment:
Reword 

Tabs allow team members to access services and content in a specific space within a meeting. This lets the team work directly with tabs and have conversations about the tools and data available within tabs. In Teams meeting, users can add a tab by selecting plus, and choosing the app that they want to install as a tab.